src/NosCore.GameObject/Providers/QuestProvider/QuestProvider.cs

```diff
@@ -132,20 +132,51 @@ private async Task<bool> IsValidScriptAsync(ICharacterEntity character, QuestAct
                     || !script.Argument1.HasValue
                     || await AddQuestAsync(character, (short)script.Argument1).ConfigureAwait(false)
                 ),
+                "time" => await TimeAsync(script.Argument1 ?? 0, character).ConfigureAwait(false),
+                "targetoff" => await TargetOffPacketAsync(script.Argument1 ?? 0, character).ConfigureAwait(false),
                 "web" => true,
                 "talk" => true,
-                "target" => true,
                 "openwin" => true,
-                "opendual" => false,
-                "time" => false,
-                "move" => false,
-                "run" => true,
-                "q_pay" => true,
-                "targetoff" => true,
+                "opendual" => true,
+
+                "move" => false, //todo handle
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5304413)